guarddog/analyzer/sourcecode/shady-links.yml

```diff
@@ -12,4 +12,5 @@ rules:
         - pattern-regex: (http[s]?:\/\/.*\.(link|xyz|tk|ml|ga|cf|gq|pw|top|club|mw|bd|ke|am|sbs|date|quest|cd|bid|cd|ws|icu|cam|uno|email|stream)\/)
     languages:
       - python
+      - javascript
```

Comment:
We should probably have strictly distinct rule files for Python and JS. Here, we're making the assumption that valid Python code using eval is also valid JS code, which is quite a stretch

Thoughts?

Reply:
For this exact rule we only look for http(s) links right?

Reply:
Ah yes sorry correct.